hivemq-edge/src/frontend/src/extensions/datahub/components/helpers/MessageInterpolationTextArea.tsx

```diff
@@ -0,0 +1,79 @@
+import { ChangeEvent, FC, FocusEvent, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { labelValue, WidgetProps } from '@rjsf/utils'
+import { getChakra } from '@rjsf/chakra-ui/lib/utils'
+import { Button, ButtonGroup, FormControl, FormLabel, Textarea, VStack } from '@chakra-ui/react'
+
+export const MessageInterpolationTextArea = (props: WidgetProps) => {
+  const { t } = useTranslation('datahub')
+  const chakraProps = getChakra({ uiSchema: props.uiSchema })
+
+  const [cur, setCur] = useState<[number, number] | undefined>(undefined)
```

Comment:
suggestion: I suggest to make it clear that it's the cursor state.

Why:
I first thought we are handling the current object or something :D


```suggestion
  const [cursor, setCursor] = useState<[number, number] | undefined>(undefined)
```